doc/source/guzzle_sphinx_theme/guzzle_sphinx_theme/static/guzzle.css_t

```diff
@@ -102,6 +111,10 @@ div.highlight {
   font-weight: bold;
 }
 
+#nav-link {
```

Comment:
Lets change this to a class `.nav-link`. id is a unique identifier for a particular html element.